ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -1050,6 +1069,29 @@ - (BOOL)shouldDisplayProgressLabel {
     return self.showsProgressInNavigationBar && [_task respondsToSelector:@selector(progressOfCurrentStep:withResult:)];
 }
 
+- (nullable ORKReviewStep*)reviewStepForStep:(ORKStep *)step {
```

Comment:
I think just create a `lastReviewStep` would be sufficient. Or we can just save the reviewStep on ORKStepVC to save the time of searching.